src/components/Home.vue

```diff
@@ -52,7 +52,7 @@
 
 <style scoped>
 #headline{
-    color:black;
-    background: linear-gradient(#149dcc, grey, transparent);
+    color: darkslategrey;
+    background: linear-gradient(greenyellow, ghostwhite, transparent);
```

Comment:
Please return this the way it is. There is a colored theme that is being used across the whole project. And the same for the border-color below.